manticore/core/manticore.py

```diff
@@ -10,6 +10,8 @@
 
 import functools
 import types
+from elftools.elf.elffile import ELFFile
+from elftools.elf.sections import SymbolTableSection
```

Comment:
Those imports along with the functionality needs to be moved to `native/manticore.py` or probably even better - to `binary/`.

Since Manticore can be used for analysing both native binaries and ethereum smart contracts we want to keep `ManticoreBase` not dependant on any of them.